### DISCUSSION TITLE
How can I run this on Fedora.

### DISCUSSION THREAD

**Discussion by pipe-organ** [sxyazi/yazi#1604] [1. Q&A]
Really loved using it on my mac. Now I want to use it in Fedora but no idea how to install it. Please help...

**Comment by AnirudhG07** [comment]
Hi! You can check out [installation](https://yazi-rs.github.io/docs/installation) instructions.
Here you can check out download yazi from source, that would be very useful.
Or if you dont want to build from source, you can download the [Nightly Version](https://github.com/sxyazi/yazi/releases/tag/nightly).
